src/bin/tikv-ctl.rs

```diff
@@ -333,16 +344,72 @@ trait DebugExecutor {
             .wait()
             .unwrap_or_else(|e| perror_and_exit("Get region id from PD", e))
         {
-            Some(region) => self.set_region_tombstone(region_id, region),
+            Some(region) => {
+                self.set_region_tombstone(region_id, region);
+                println!("success!");
+            }
             None => {
                 eprintln!("no such region in pd: {}", region_id);
                 process::exit(-1);
             }
         }
     }
 
+    fn transfer_leader_for_node_recovery(&self, region_id: u64, endpoints: Vec<String>) {
+        self.check_local_mode();
+        match RpcClient::new(&endpoints)
+            .unwrap_or_else(|e| perror_and_exit("RpcClient::new", e))
+            .get_region_and_leader_by_id(region_id)
+            .wait()
+            .unwrap_or_else(|e| perror_and_exit("Get Region from PD", e))
+        {
+            Some((region, leader)) => {
+                let store_id = self.get_store_id();
+                let new_leader = region
+                    .get_peers()
+                    .into_iter()
+                    .find(|p| {
+                        let p_store_id = p.get_store_id();
+                        p_store_id != store_id && p_store_id != leader.get_store_id()
+                    })
+                    .expect("Cant get new leader");
+
+                #[derive(Serialize)]
+                struct Body {
+                    name: &'static str,
+                    region_id: u64,
+                    to_store_id: u64,
+                }
+
+                let url = format!("http://{}/pd/api/v1/operators", endpoints[0]);
```

Comment:
seem we should send to PD leader

Reply:
I have tested this, we can send request to followers, and followers will redirect it to leader.